src/ImageSharp/Formats/Png/PngEncoder.cs

```diff
@@ -42,6 +42,16 @@ public sealed class PngEncoder : IImageEncoder, IPngEncoderOptions
         /// <inheritdoc/>
         public PngInterlaceMode? InterlaceMethod { get; set; }
 
+        /// <summary>
+        /// Gets or sets the chunk filter. This can be used to exclude some ancillary chunks from being written.
+        /// </summary>
+        public PngChunkFilter? ChunkFilter { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether fully transparent pixels should be converted to black pixels.
+        /// </summary>
+        public bool MakeTransparentBlack { get; set; }
```

Comment:
I don't really understand the use case for this property. Why would we choose `Black` over `Transparent`?

Reply:
@JimBobSquarePants the intention is to convert fully transparent pixels that may contain r g b values to a transparent black I.e r g b of 0 and thus should six compression

Reply:
So bane maybe confusing but something like MakeTransparentIntoTransparentBlack Is long :)

Reply:
@UkooLabs  That's what I thought it meant to do. However the implementation is converting to `Color.Black` rather than `Color.Transparent`. Probably because the property name needs some love. `ClearTransparency` perhaps or even a two property enum. `PngTransparentColorBehavior` `Clear`, `Preserve`

Reply:
yeah i really misunderstood that. I was thinking it should be `Color.Black`

Reply:
Naming is hard...